Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,16 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const durationInHours = tasks.map(minute => minute.duration / 60);
```

Comment:
If you map an array of `tasks` then an individual element of that array is a `task`, not a `minute`. So this line would be more accurate if it read like this:

```js
const durationsInHours = tasks.map(task => task.duration / 60);
```

Note that I have added an 's` after 'duration' in the name `durationsInHours` to make explicit that we get back an array.